src/pyEQL/utils.py

```diff
@@ -32,6 +32,29 @@ def standardize_formula(formula: str):
     return Ion.from_formula(formula).reduced_formula
 
 
+def format_solute(solute_dict: dict, units: str):
+    """
+    Format a solute dictionary with the given units.
+
+    Args:
+        solute_dict: The dictionary to format.
+        units: The units to use for the solute.
+
+    Returns:
+        A formatted solute dictionary.
+
+    Raises:
+        TypeError if `solute_dict` is invalid.
```

Comment:
Can you please make a few edits to the docstring?

1. I suggest the name `format_solutes_dict` because the method formats a dictionary, not an individual solute, and because the kwarg in `Soluteion.__init__` is called `solutes` not `solute`
2. Please be more explicit about what the method does and what the arguments are, e.g. "Formats a dictionary of solute: amount" (where amount is a number) by converting the amount to a str with units, suitable for passing to `Solution.__init__`. Note that all solutes must be given in the same units"
3. For the `solute_dict` arg description, please state how the solute dict should be formatted (e.g., keys should be X, values should be Y